README.md

```diff
@@ -4,97 +4,164 @@
+Send the following Slack message to Atomist:
+```
+update boosters maven property
+```
+
+You will need to provide a git branch, property name and a new value of the property.
+
+
+### Create a launcher pull request
 
-Of particular importance is the `Minimal Configuration` section which mentions that at the very least, a file named `$HOME/.atomist/client.config.json`
-is needed containing the following information
+This command updates our fork of the Launcher catalog and raises a pull request upstream.
 
+Send the following Slack message to Atomist:
 ```
-{
-  "apiKey": "API_KEY",
-  "workspaceIds": [
-    "WORKSPACE_ID"
-  ]
-}
+create launcher pr
 ```
 
-with API_KEY and WORKSPACE_ID replaced with your Atomist API key and workspace ID, respectively. 
+You will need to provide a Spring Boot version that should be used by the Launcher (e.g. 2.3.0.RELEASE). The script will figure out which example tags to use based on that version and will update the Launcher accordingly.
 
-* Node 8.x or higher
+### Update an example
 
-Install dependencies by executing:
+TODO
 
-`npm install`
+### Generate licenses
 
-### Launch application
+This command generates licenses for a Maven project using [Licenses Generator](https://github.com/snowdrop/licenses-generator) and stores them in a `src/licenses` directory.
 
-Before launching the application the following environment variables need to be set:
+Send the following Slack message to Atomist:
+```
+generate licenses
+```
 
-| Name        | Purpose           | Mandatory? |
-| ------------- |:-------------:| -----: |
-| SENDGRIDKEY      | Sendgrid api key needed for sending email | No |
+You will need to provide a repository's owner, name and branch.
 
-Launch application by issuing:
+### Release examples
 
-`npm run clean && npm run compile && npm start`
+This command releases all the Snowdrop examples. The release entails process entails the following actions:
+1. Creating two tags - one with community release and one with prod release.
+1. Generating licenses for each of the created tags (see [Generate licenses](#generate-licenses))
+1. Bumping a project version in a branch on which this release is based on.
 
-## Install specific nvm version
+Send the following Slack message to Atomist:
+```
+release examples
+```
 
-To manage specific node versions, as the application requires Node 8.x, install `nvm` (Node Version Manager), instructions [here](https://github.com/nvm-sh/nvm).
+You will need to provide a Spring Boot version to which this release is tied (e.g. 2.3.0.RELEASE) and a prod BOM version of this release (e.g. 2.3.0.Final-redhat-00001).
 
-With `nvm` installed, use it install the latest 8.x node version.
+### Release single example
 
-```bash
-$ nvm install 8
+This command is identical to the [Release examples](#release-examples) except it released only a single example.
```

Comment:
except it released only --> except it releases only

Reply:
```suggestion
This command is identical to the [Release examples](#release-examples) except it simply releases a single example.
```